### ISSUE TITLE
implement particle system

### ISSUE THREAD

**Issue by philiparvidsson** [philiparvidsson/Sap6-Game#60]
the scope of this card is to implement a 3-dimensional particle system simulating at least life time and solving differential equations for individual particle positions.

when done, further functionality might be scoped in (or open another issue)

**Comment by philiparvidsson** [comment]
blocked on #59

**Comment by philiparvidsson** [comment]
blocked no #68

**Comment by philiparvidsson** [comment]
https://github.com/philiparvidsson/Sap6-Game/commit/00cbb27dcf9d189c4549102cd8591f7c52229009

**Comment by philiparvidsson** [comment]
see following for an example on how to spawn particles:

https://github.com/philiparvidsson/Sap6-Game/blob/master/Source/GameName/Dev/CollisionPlayground.cs#L74-L82
